aspnet-core/xml/Microsoft.AspNetCore.ApiAuthorization.IdentityServer/ApiAuthorizationDbContext`1.xml

```diff
@@ -197,7 +204,7 @@
         <AssemblyVersion>3.1.18.0</AssemblyVersion>
         <AssemblyVersion>5.0.9.0</AssemblyVersion>
         <AssemblyVersion>6.0.6.0</AssemblyVersion>
-        <AssemblyVersion>7.0.0.0</AssemblyVersion>
+        <AssemblyVersion>7.0.3.0</AssemblyVersion>
```

Comment:
Shouldn't this have been 7.0.5.0 also?

Reply:
No. The IdentityServer4.EntityFramework.Interfaces.IPersistedGrantDbContext.SaveChangesAsync member does not exist in the library in the Microsoft.AspNetCore.ApiAuthorization.IdentityServer version 7.0.5 NuGet package.

Reply:
There wouldn't be any public API changes in a patch release. `Microsoft.AspNetCore.ApiAuthorization.IdentityServer` has not referenced `IdentityServer4.EntityFramework.Interfaces.IPersistedGrantDbContext.SaveChangesAsync` since 6.0-rc1 when the reference switched to `Duende.IdentityServer.EntityFramework.Interfaces.IPersistedGrantDbContext.SaveChangesAsync` by https://github.com/dotnet/aspnetcore/pull/34321. Where is 7.0.3.0 coming from?